src/API/Google/AdsCampaignCriterion.php

```diff
@@ -35,6 +55,35 @@ function ( $location_id ) use ( $campaign_resource_name ) {
 			$location_ids
 		);
 	}
+	/**
+	 * Returns a set of operations to delete multiple campaign criteria.
+	 *
+	 * @param string $campaign_resource_name
+	 * @return array
+	 */
+	public function delete_operations( string $campaign_resource_name ): array {
+		$operations = [];
+
+		$results = ( new AdsCampaignCriterionQuery() )
+			->set_client( $this->client, $this->options->get_ads_id() )
+			->where( 'campaign_criterion.campaign', $campaign_resource_name )
+			->where( 'campaign_criterion.negative', 'false', '=' )
+			->where( 'campaign_criterion.status', 'REMOVED', '!=' )
```

Comment:
I'm not able to see the status either in the Google dashboard, should we also remove it here?